distribution/docs/src/main/resources/content/_installing/java-reqs.adoc

```diff
@@ -77,6 +88,51 @@ echo %JAVA_HOME%
 ----
 ====
 
+If JRE was installed:
+====
+.Setting JRE_HOME variable
+Replace `<JAVA_VERSION>` with the version and build number installed.
+
+. Open a terminal window(*NIX) or command prompt (Windows) with administrator privileges.
+. Determine Java Installation Directory (This varies between operating system versions).
++
+.Find Java Path in *NIX
+----
+which java
+----
++
+.Find Java Path in Windows
+The path to the JRE can vary between versions of Windows, so manually verify the path under:
++
+----
+C:\Program Files\Java\jre<M.m.p_build>
+----
++
+. Copy path of Java installation to clipboard. (example: `/usr/java/<JAVA_VERSION`>)
+. Set `JRE_HOME` by replacing <PATH_TO_JAVA> with the copied path in this command:
++
+.Setting `JRE_HOME` on *NIX
+----
+JRE_HOME=<PATH_TO_JAVA><JAVA_VERSION>
+export JRE_HOME
+----
++
+.Setting `JRE_HOME` on Windows
+----
+set JRE_HOME=<PATH_TO_JAVA><JAVA_VERSION>
+setx JRE_HOME "<PATH_TO_JAVA><JAVA_VERSION>"
+----
++
+.Adding `JRE_HOME` to `PATH` Environment Variable on Windows
```

Comment:
❓ Is adding `JRE_HOME` to the `PATH` environment variable not necessary on *NIX systems?

Reply:
i need to get clarification. There was no mention in the docs currently for `JAVA_HOME` I just copied instructions and changed JAVA to JRE

Reply:
added